redfish-core/lib/assembly.hpp

```diff
@@ -106,6 +106,86 @@ inline void
                 std::array<const char*, 0>{});
         }
 
+        // Special handling for LCD and base panel CM.
+        if (sdbusplus::message::object_path(assembly).filename() == "panel1" ||
```

Comment:
Is there a reason why this code that is setting the ReadyToRemove property isn't handled where we otherwise set the present state? line 407

Reply:
I put up this code here, below tod_battery CM to have similar functionalities together but overlooked that this can be moved to line 407 which will save us couple of DBus calls and also avoid redundant code as we are already getting present property there.

Updated the code accordingly.